airflow/contrib/secrets/hashicorp_vault.py

```diff
@@ -142,6 +142,12 @@ def client(self):
             scopes = _get_scopes(self.gcp_scopes)
             credentials, _ = get_credentials_and_project_id(key_path=self.gcp_key_path, scopes=scopes)
             _client.auth.gcp.configure(credentials=credentials)
+        elif self.auth_type == "gce":
```

Comment:
Because this is pretty Sift-specific (mostly related to getting the labels), can we call this `sift_gce`?

Reply:
agree, fixed.